app/src/main/res/values/styles.xml

```diff
@@ -1,15 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources xmlns:android="http://schemas.android.com/apk/res/android">
+<resources
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:android="http://schemas.android.com/apk/res/android">
 
     <!-- Main theme -->
 
-    <style name="Base.AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
+    <style name="AppBaseTheme" parent="Theme.AppCompat.Light.NoActionBar">
+        <!-- Changes specific to pre-v21 versions of Android -->
+    </style>
+
+    <style name="AppTheme" parent="AppBaseTheme">
```

Comment:
It keeps the name of the primary theme smaller 'AppTheme', and the sample Android projects in v14 (when first Android introduced this concept of having different base themes in different platforms, and finally inheriting them into a AppTheme), used to have it set up this way, explicitly mentioning a parent.